lib/helpers.bash

```diff
@@ -115,6 +118,27 @@ _bash-it-plugins ()
     _bash-it-describe "plugins" "a" "plugin" "Plugin"
 }
 
+_bash-it_update() {
+  _about 'updates Bash it'
+  _group 'lib'
+
+  cd "${$BASH_IT}"
```

Comment:
I think the second "$" is wrong. I get the following error when running this:

    -bash: ${$BASH_IT}: bad substitution

When I remove the second "$" (`${BASH_IT}`), it works fine for me.